### PR TITLE
Add Babylon.JS icon

### PR DESCRIPTION
**Pull request by jglamp**
Babylon js logo

**Issue:** closes #9980

**Popularity metric:** 22.6K stars on GitHub

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

Got permission to readd this in #9980.

### CODE REVIEW

File: icons/babylondotjs.svg
Comment (adamrusted) [inline icons/babylondotjs.svg:1]: Looks as though there are some excess paths in this - can sometimes happen when using certain tools. Could you have a look at removing them please, @jglamp?
image

**Comment by jglamp** [comment]
Alright, it looked like some shapes weren't fully merged. I got rid of a few nodes and it looks now to be all one path.

**Review by adamrusted — APPROVED** [review]
It's a perfect match now - thanks @jglamp!

**Comment by jglamp** [comment]
Fantastic! Thanks so much!

**Pull request merged**